src/lib/actions/projects.ts

```diff
@@ -29,15 +31,26 @@ export const getProjects = async (farcasterId: string) => {
 export const createNewProject = async (details: CreateProjectParams) => {
   const session = await auth()
 
-  if (!session?.user?.id) {
+  if (!session?.user?.id || !session.user.farcasterId) {
```

Comment:
Oh should we just have the projection created under id here for consistency?

Reply:
🫡 will do in a follow-up